netsuite/10.0/modules/ROOT/pages/netsuite-call-restlets-example.adoc

```diff
@@ -0,0 +1,159 @@
+}
+
+// Create a standard NetSuite record
+
+function createRecord(datain)
+{
+    var err = new Object();
+
+    // Validate if mandatory record type is set in the request
+    if (!datain.recordtype)
+    {
+        err.status = "failed";
+        err.message= "missing recordtype";
+        return err;
+    }
+
+    var record = nlapiCreateRecord(datain.recordtype);
+
+    for (var fieldname in datain)
+    {
+      if (datain.hasOwnProperty(fieldname))
+      {
+        if (fieldname != 'recordtype' && fieldname != 'id')
+        {
+          var value = datain[fieldname];
+            if (value && typeof value != 'object')
+            // ignore other type of parameters
+            {
+              record.setFieldValue(fieldname, value);
+            }
+        }
+      }
+    }
+    var recordId = nlapiSubmitRecord(record);
+    nlapiLogExecution('DEBUG','id='+recordId);
+    var nlobj = nlapiLoadRecord(datain.recordtype,recordId);
+    return nlobj;
+}
+
+// Delete a standard NetSuite record
+function deleteRecord(datain)
+{
+  nlapiDeleteRecord(datain.recordtype, datain.id);
+  // for example: recordtype="customer", id="769"
+
+}
+----
++
+. Enable SuiteScript and NetSuite Web Services in your account. 
+.. Log in to NetSuite.
+.. Click *Set Up > Company > Enable Features > SuiteCloud*.
++
+image::netsuite-enable-suitescript.png[Enable SuiteScript]
++
+. Create a new script and upload the file you <<deploy-a-script-as-restlet,previously created>>.
+.. Go to *Customization > Scripting > Scripts > New*.
+.. Select the script file, click *Create Script Record*, and select *RESTlet*.
++
+image::netsuite-script-type.png[Select Script Type]
++
+. Fill out the form using the content of the script you uploaded, and deploy the script.
++
+image::netsuite-setup-script.png[Setup Script]
++
+. After selecting your audience you see the following page:
++
+image::netsuite-script-deployment.png[Script Deployment]
++ 
+Take note of the script and deploy numbers in the external URL as you will need them later to call the RESTlet.
+
+== Import the Project
+
+* Go to *File > Import*.
+* From the Anypoint Studio parent folder, select *Anypoint Studio Project > External Location*.
+* Provide the root path to the demo project folder.
+* Select Mule Runtime (4.1.1 or later).
+* Click *Finish*.
+* Configure the credentials, as well as the deployed script and deploy IDs, inside the file `src/main/resources/mule-app.properties`.
++
+[source,javascript,linenums]
+----
+netsuite.email=
+netsuite.password=
+netsuite.account=
+netsuite.roleId=
+netsuite.applicationId=
+netsuite.subsidiary=
+netsuite.script=
+netsuite.deploy=
+----
++
+. Open the Global Element Configuration.
+. Click *Test Connection* to ensure there is connectivity with the sandbox.
++
+Testing the connection is a crucial step. If you experience connectivity issues, you won't be able to execute any part of this example, nor make use of DataSense.
++
+. A successful message should appear:
```

Comment:
I don't think this is essential. I would assume connectivity is required to execute.
And the message appearing is not a step.
So I would end the previous with a colon and just show the message as the result.